assets/blocks/editor-components/toolbar-dropdown/index.js

```diff
@@ -0,0 +1,78 @@
+import {
```

Comment:
It's part of another PR: https://github.com/Automattic/sensei/pull/3904

Reply:
Should we hold off on review until #3904 is merged? Or just ignore this file when reviewing?

Reply:
#3904 merged! =)